src/lib/API/api.js

```diff
@@ -531,7 +540,14 @@ export class APIService {
       })
 
       params.page += 1
-      txs.push(...events)
+
+      if (isArray(events)) {
+        txs.push(...events)
+      } else {
```

Comment:
1. do not need, just if (!isArray(events)) { throw error }
2. you could just log.warn { events, params, chainId } here, do not add data if it's only needs for logging. in FV we used errors with custom .name props because it was used in the FV flow